2D Robot Game/Assets/Scripts/PlayerController.cs

```diff
@@ -0,0 +1,43 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerController : MonoBehaviour
+{
+    private float horizontalInput;
+    private float verticalInput;
+
+    public float _speed = 10.0f;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        MovePlayer();
+        LookAtMouse();
+
+    }
+
+    private void MovePlayer()
+    {
+        // Gets input from player
+        Vector2 input = new Vector2(x: Input.GetAxis("Horizontal"), y: Input.GetAxis("Vertical"));
```

Comment:
maybe we could normalize this vector so that the velocity is always the same even if the player is moving diagonally.
```suggestion
        Vector2 input = new Vector2(x: Input.GetAxis("Horizontal"), y: Input.GetAxis("Vertical")).normalized;
```

Reply:
Thanks for the suggestion! I just tested it out, and I saw that the normalized vector causes the movement to be kind of choppy.  Sometimes, it feels that the player doesn't stop after I let go of the button. Do you have an idea on how to fix that?

Reply:
You are right, it moves a little weird. Not sure how to fix that 😿

Reply:
maybe we should ignore the normalized for now

Reply:
Sure, let's do that